README.md

```diff
@@ -1,9 +1,7 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+## Environment variables
+### Creating the databases
+We will have two databases in this project- one for development data, and another scaled down data for testing.
 
-
-
---- 
-
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+Create two .env files - .env.test and .env.development. In each of the files, add PGDATABASE=_database-name_. Replace _database-name_ with the correct database name depending on the environment (i.e development or test). Ensure that both the .env files are .gitignored.
```

Comment:
What are the database names? Or where can they be found? 

The final bit of adding this to .gitignored shouldnt be here. That is for you to do and once that is done it wont need to be done again.